apps/condo/domains/billing/schema/ValidateQRCodeService.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Generated by `createservice billing.ValidateQRCodeService --type mutations`
+ */
+const { isEmpty } = require('lodash')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT, INTERNAL_ERROR } } = require('@open-condo/keystone/errors')
+const { GQLCustomSchema } = require('@open-condo/keystone/schema')
+
+const { AcquiringIntegrationContext } = require('@condo/domains/acquiring/utils/serverSchema')
+const { BankAccount } = require('@condo/domains/banking/utils/serverSchema')
+const access = require('@condo/domains/billing/access/ValidateQRCodeService')
+const { BILLING_VALIDATE_QR_CODE_WINDOW, MAX_CLIENT_VALIDATE_QR_CODE_BY_WINDOW } = require('@condo/domains/billing/constants')
+const { WRONG_FORMAT, NOT_FOUND } = require('@condo/domains/common/constants/errors')
+const { CONTEXT_FINISHED_STATUS } = require('@condo/domains/miniapp/constants')
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+const { RedisGuard } = require('@condo/domains/user/utils/serverSchema/guards')
+
+const redisGuard = new RedisGuard()
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const ERRORS = {
+    INVALID_QR_CODE: {
+        mutation: 'validateQRCode',
+        variable: ['data', 'qrCode'],
+        code: BAD_USER_INPUT,
+        type: WRONG_FORMAT,
+        message: 'Provided QR code doesn\'t have one of required fields: BIC, payerAddress, lastName, paymPeriod or Sum',
```

Comment:
No translated errors? Can this error be shown to the user?

Reply:
I do not think that we should explain something about qr-code requisites to the user who scanned the code.